tests/test_strategy_load_hours.py

```diff
@@ -433,6 +433,11 @@ def test_balancing_offers_are_created_if_device_in_registry(
 def test_use_market_maker_rate_parameter_is_respected(use_mmr, expected_rate):
     GlobalConfig.market_maker_rate = 9
     load = LoadHoursStrategy(200, final_buying_rate=33, use_market_maker_rate=use_mmr)
+
+    load.area = MagicMock(spec=Area)
+    load.area.now = now()
```

Comment:
I assume that you need the 'now' area property just because it is used in the activate method, right ? Can a mock not work instead of the pendulum function ?

Reply:
Yeah, I could mock it. Can you tell me the advantage? Is mocking using less resources?

Reply:
Well, in this specific case it does not hurt of course, but in general it is an antipattern to use the current time in unit tests. The reason for that is because there is no way to reproduce the exact same test run in 2 different now calls, because the different test runs will take place in different points in time. This makes the tests way less predictable (e.g. one test might fail today but succeed tomorrow, which makes the tests unpredictable). I would like to repeat though, that this is not such a case, therefore feel free to leave it as is if you prefer it.

Reply:
Thanks for the extensive explanation! Totally makes sense. Will change this here (and on the other occasions) .